server/src/main/java/org/elasticsearch/index/query/ExistsQueryBuilder.java

```diff
@@ -181,6 +192,45 @@ private static Query newObjectFieldExistsQuery(QueryShardContext context, String
         return new ConstantScoreQuery(booleanQuery.build());
     }
 
+    /**
+     * Helper method to validate whether field is mapped
+     * @return true if the field is mapped, false otherwise
+     */
+
```

Comment:
please remove this empty line for consistency with the rest of the code base, we generally don't leave empty lines between javadocs and methods

Reply:
yes made changes.